Prefabs/Blue_Gem.prefab

```diff
@@ -157,7 +157,6 @@
                     "Id": 4709323619022222983,
                     "ColliderConfiguration": {
                         "Trigger": true,
-                        "InSceneQueries": false,
```

Comment:
Why did these get removed from the gem prefabs? This will reintroduce the bug of the camera colliding with the gems.

Reply:
This is accidental, not sure how it happened. these re easy to fix.